src/Editor/index.js

```diff
@@ -77,8 +77,7 @@ class WysiwygEditor extends Component {
     const newState = {};
     const { toolbar, editorState, contentState } = this.props;
     if (toolbar !== prevProps.toolbar) {
```

Comment:
@jpuri those objects are always different when user type inside the editor. Hence, we merge every render. Could you explain why we need to merge here?

Reply:
Hey @nlopin : you are right I fixed this issue. Thanks 👍